.github/workflows/tests.yaml

```diff
@@ -0,0 +1,48 @@
+name: gabbi tests
```

Comment:
not sure whether that's a sensible name - if only because it shows up in URLs:
```
.. image:: https://github.com/cdent/gabbi/workflows/gabbi%20tests/badge.svg
    :target: https://github.com/cdent/gabbi/actions
```

Reply:
Is `tests` (by itself) too subtle?

Reply:
no, I prefer that too, now that I understand the context of where it shows up (e.g. the badge); will do